src/components/Wizard/Wizard.tsx

```diff
@@ -102,6 +102,27 @@ export default function Wizard<T>({
 		history.push(basePath);
 	}
 
+	// This is a dummy method extracted from the original wizard implementation
+	function clickToCancel() {
+		onCancel({ file, metadata }).catch((err) => {
+			// TODO handle error gracefully
+			console.error(err);
+		});
+		resetAndClose();
+	}
+
+	// This is a dummy method extracted from the original wizard implementation
+	function clickToNextStep() {
```

Comment:
Functions are relatively cheap but will get re-created on each re-render. Arguably there is no benefit for having these function separated from the callback prop.
I'd typically recommend to split it out if the re-creation of the function is complex enough that the use of React.useCallback is justified or in case it is used more than once. If it's a one liner I'd leave it inline. 
Arguable clickToCancel above should have used useCallback as well. However it is a trade-off between increased performance and increased complexity. In this instance I chose to keep it simple..

Reply:
I've reverted it to an inline script. I extracted the methods, because I could imagine that `wizard.tsx` would act like a controller method and thus these methods would become more complex eventually. But for now, reverted it.